docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -93,6 +93,8 @@ If `true`, the snapshot's persistent settings, index templates, ingest
 pipelines, and {ilm-init} policies are restored into the current cluster. This
 overwrites any existing cluster settings, templates, pipelines and {ilm-init}
 policies whose names match those in the snapshot.
+<2> Regular expression that will match the name of data streams and indices in the snapshot. This supports https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#cg-[`capturing groups`].
```

Comment:
```suggestion
<2> Regular expression that matches data streams and indices in the snapshot.
Supports
https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#cg-[`capturing
groups`].
```

Reply:
+1 to @jrodewig's changes. 

@romain-chanu, I'm curious why we're calling out capturing groups specifically. The page for the `Pattern` class includes everything about the syntax, so I think we might want to say something like:
>Supports https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#cg-[regular expression constructs].

Please let me know if there was reason to call out capturing groups.

Reply:
Thank you @jrodewig @lockewritesdocs - thank you for the suggestions - that looks good. I wanted to mention the capturing groups explicitly to better explain the example - but we can definitely mention the page for the `Pattern` (and @jrodewig's suggestion for point 3 still refers to the capturing group) so i think we are good.